.github/workflows/ci.yml

```diff
@@ -17,6 +17,9 @@ jobs:
   quality:
     name: Check quality
     runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      pull-requests: write
```

Comment:
https://docs.github.com/en/actions/using-jobs/assigning-permissions-to-jobs#overview